easybuild/easyconfigs/c/CapnProto/CapnProto-1.0.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# Contribution by
+# DeepThought, Flinders University
+# Updated to 0.9.1
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'ConfigureMake'
+
+name = 'CapnProto'
+version = '1.0.1'
+
+homepage = 'https://capnproto.org'
+description = "Cap’n Proto is an insanely fast data interchange format and capability-based RPC system."
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+# -std=c++17 to avoid problems like "error: 'aligned_alloc' was not declared in this scope"
+# lowopt (-O1) to avoid problems like "capnp/schema-loader.c++:1971: failed: no schema node loaded for ..."
+# (only on Intel Skylake?)
+toolchainopts = {'cstd': 'c++17', 'lowopt': True}
```

Comment:
```suggestion
```

I've built on IceLake and CascadeLake without the `lowopt` and not seen the mentioned failure.

From https://gcc.gnu.org/projects/cxx-status.html#cxx17
> C++17 mode is the default since GCC 11

I've opened #18942 to remove these from the GCC 11.x CapnProto easyconfigs.